extensions/amp-story/1.0/amp-story-bling-link.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Bling link component that turns into a tooltip when clicked.
+ */
+
+import {htmlFor} from '../../../src/static-template';
+
+/**
+ * Links that are bling links.
+ * @const {string}
+ */
+export const BLING_LINK_SELECTOR = 'a.i-amphtml-story-bling-link';
```

Comment:
Publishers won't be able to use this for now as this is invalid AMP, we can change it later once the feature is ready :)